src/elements/content-sidebar/stories/__mocks__/MetadataSidebarRedesignedMocks.ts

```diff
@@ -0,0 +1,323 @@
+export const fileId = '415542803939';
+export const fileIdWithoutMetadata = '416047501580';
+
+export const mockFileRequest = {
+    url: `https://api.box.com/2.0/files/${fileId}?fields=is_externally_owned,permissions`,
+    response: {
+        type: 'file',
+        id: fileId,
+        etag: '3',
+        is_externally_owned: false,
+        permissions: {
+            can_download: true,
+            can_preview: true,
+            can_upload: true,
+            can_comment: true,
+            can_rename: false,
+            can_delete: false,
+            can_share: false,
+            can_set_share_access: false,
+            can_invite_collaborator: false,
+            can_annotate: false,
+            can_view_annotations_all: true,
+            can_view_annotations_self: true,
+            can_create_annotations: true,
+            can_view_annotations: true,
+        },
+    },
+};
+
+export const mockFileRequestWithoutMetadata = {
+    url: `https://api.box.com/2.0/files/${fileIdWithoutMetadata}?fields=is_externally_owned,permissions`,
+    response: {
+        type: 'file',
+        id: fileIdWithoutMetadata,
+        etag: '3',
+        is_externally_owned: false,
+        permissions: {
+            can_download: true,
+            can_preview: true,
+            can_upload: false,
+            can_comment: true,
+            can_rename: false,
+            can_delete: false,
+            can_share: false,
+            can_set_share_access: false,
+            can_invite_collaborator: false,
+            can_annotate: false,
+            can_view_annotations_all: true,
+            can_view_annotations_self: true,
+            can_create_annotations: true,
+            can_view_annotations: true,
+        },
+    },
+};
+
+export const mockEmptyMetadataInstances = {
+    url: `https://api.box.com/2.0/files/${fileIdWithoutMetadata}/metadata`,
+    response: {
+        entries: [],
+    },
+};
+
+export const mockMetadataInstances = {
+    url: `https://api.box.com/2.0/files/${fileId}/metadata`,
+    response: {
+        entries: [
+            {
+                $id: '776397f2-5d56-4a33-80c1-569614702899',
+                $version: 1,
+                $type: 'myTemplate-453ffc2f-bf5a-464c-a004-476a4eac20fd',
+                $parent: 'file_415542803939',
+                $typeVersion: 1,
+                $template: 'myTemplate',
```

Comment:
Shouldn't this be `templateKey`?

Reply:
No, https://developer.box.com/reference/resources/metadata--full/#param-$template